src/holodeck/spaces.py

```diff
@@ -34,6 +34,18 @@ def shape(self):
         """
         return self._shape
 
+    def get_low(self):
+        """
+        Returns: the action space's minimum value(s)
```

Comment:
We use Google style docstrings. It would be helpful to include the type here
```
Returns:
    :obj:`list` of :obj:`float` or :obj:`float`: The action space's minimum value(s)
```

Reply:
Done